### PR TITLE
refactor: use runAtMidnight function to loop

### PR DESCRIPTION
**Pull request by bpleonardo**
Hi, I made `reset.js` an arrow function in `bot.js` because it was only 2 lines of code.

I can restore it (and do the required changes), if you wish.

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -8,8 +8,10 @@ require('dotenv').config();
 const client = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES, Intents.FLAGS.GUILD_MESSAGE_REACTIONS] });
 // importa as funções de database
 const { checkDatabase } = require('./functions/database.js');
-// importa o loop para checar o horário
-const { loopUtilMidnight } = require('./functions/reset.js');
+// importa a função para trocar a palavra
+const { newWord } = require('./database.js');
```
Comment (vvito7) [inline src/bot.js]:
```suggestion
const { newWord } = require('./functions/database.js');
```
Wrong path. 🙃

File: src/bot.js [hunk 2]
```diff
@@ -8,8 +8,10 @@ require('dotenv').config();
...
+// importa a função para iniciar o timer
+const { runAtMidnight } = require('./functions/runner');
```
Comment (vvito7) [inline src/bot.js]:
```suggestion
const { runAtMidnight } = require('./functions/runner.js');
```
Even if it works, it is better to also put the extension.

File: src/commands/adivinhar.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ const readline = require('readline');
 
 const { checkWordDatabase, checkUserDatabase, itPlayed, getDayNumber } = require('../functions/database.js');
 const { square, letter } = require('../utils/emotes.json');
+const { runAtMidnight } = require('../functions/runner');
```
Comment (vvito7) [inline src/commands/adivinhar.js]:
```suggestion
const { runAtMidnight } = require('../functions/runner.js');
```

**Review by vvito7 — CHANGES_REQUESTED** [review]
Nice catch!

File: src/bot.js (hunk 1 above)
Comment (bpleonardo) [reply src/bot.js]: Oops didn't see that

**Review by vvito7 — APPROVED** [review]
Looks good to me.

**Pull request merged**